package.json

```diff
@@ -33,25 +33,25 @@
     "lint": "eslint src"
   },
   "dependencies": {
-    "@babel/helper-module-imports": "7.0.0-beta.49",
+    "@babel/helper-module-imports": "7.0.0-beta.51",
     "rollup-pluginutils": "^2.0.1"
   },
   "peerDependencies": {
     "@babel/core": "7 || ^7.0.0-beta"
   },
   "devDependencies": {
-    "@babel/core": "7.0.0-beta.49",
-    "@babel/plugin-external-helpers": "7.0.0-beta.49",
-    "@babel/plugin-proposal-decorators": "7.0.0-beta.49",
-    "@babel/plugin-transform-runtime": "7.0.0-beta.49",
-    "@babel/preset-env": "7.0.0-beta.49",
-    "buble": "^0.18.0",
-    "eslint": "^3.12.0",
-    "mocha": "^3.2.0",
-    "rollup": "^0.52.2",
-    "rollup-plugin-buble": "^0.18.0",
-    "source-map": "^0.5.6",
-    "source-map-support": "^0.4.6"
+    "@babel/core": "^7.0.0-beta.51",
```

Comment:
it would be generally more safe to pin those beta versions

Reply:
Fixed. Thanks for taking a look.